content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +168,64 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+For every `dvc.yaml` file, a matching `dvc.lock` (YAML) file usually exists.
+It's created or updated by DVC commands such as `dvc run` and `dvc repro`.
+`dvc.lock` describes the latest pipeline state. It has several purposes:
+
+- Tracking of intermediate and final results of a pipeline — similar to
+  [`.dvc` files](#dvc-files). This is done with a mapping of file and directory
+  paths in the <abbr>project</abbr> to their locations (hash values) in the
+  <abbr>cache</abbr> or [remote storage](/doc/command-reference/remote).
```

Comment:
I wonder if this explanation is too long. Could remove the whole part about mapping between locations (hash values) to cache or remote storage. Although it's important and not sure it's stated this clearly in any other doc...

Reply:
yes, I would say that it's a similar mechanism and we can do the explanation somewhere else - e.g. we have a cache structure section somwhere

Reply:
OK, removing.